process/smartContract/scrCommon/common.go

```diff
@@ -0,0 +1,75 @@
+package scrCommon
```

Comment:
why dumping everything in common packages/files? we could make small components from what we implement

Reply:
This is a very small package with 1 small file. Was necessary because of a circular dependency headache